src/index.js

```diff
@@ -0,0 +1,28 @@
+import { createTransform } from "redux-persist";
+import LZ from "lz-string";
+import stringify from "json-stringify-safe";
+import process from "process";
```

Comment:
I'm not so sure about that in browser environment. I had some problem with `process` (and also `env`) being undefined in some browsers.